backend/server.js

```diff
@@ -0,0 +1,247 @@
+var express = require('express'); 
+var app = express(); 
+  
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/HizanLi/CPEN321/pullRequest?prid=9816566)
```suggestion

```